.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
 	"editor.formatOnSave": true,
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
+		"source.fixAll.eslint": "explicit"
```

Comment:
VSCode true is set to "explicit" since VSCode 1.85.0. It seems boolean values were supported until 1.84.0.

References

https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto https://code.visualstudio.com/updates/v1_83#_code-actions-on-save-and-auto-save